### ISSUE TITLE
[YouTube] youtube-dl terminates/crashes with KeyError: u'preference'

### ISSUE THREAD

**Issue by BlackPropeller** [rg3/youtube-dl#4648]
See below.

YouTube URL has this issue: `http://www.youtube.com/watch?v=qy2Z2mhIkQ8`
Command I ran: `./youtube-dl -g -s  http://www.youtube.com/watch?v=qy2Z2mhIkQ8 --verbose`

Output:
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-g', '-s', 'http://www.youtube.com/watch?v=qy2Z2mhIkQ8', '--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.01.05
[debug] Python version 2.6.6 - Linux-3.10.23-xxxx-std-ipv6-64-x86_64-with-centos-6.5-Final
[debug] exe versions: ffmpeg 0.6.5, ffprobe 0.6.5
[debug] Proxy map: {}
Traceback (most recent call last):
  File "/usr/lib64/python2.6/runpy.py", line 122, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.6/runpy.py", line 34, in _run_code
    exec code in run_globals
  File "./youtube-link/__main__.py", line 19, in <module>
  File "./youtube-link/youtube_dl/__init__.py", line 361, in main
  File "./youtube-link/youtube_dl/__init__.py", line 351, in _real_main
  File "./youtube-link/youtube_dl/YoutubeDL.py", line 1165, in download
  File "./youtube-link/youtube_dl/YoutubeDL.py", line 592, in extract_info
  File "./youtube-link/youtube_dl/extractor/common.py", line 245, in extract
  File "./youtube-link/youtube_dl/extractor/youtube.py", line 1049, in _real_extract
KeyError: u'preference'
```

**Comment by phihag** [comment]
Thank you for the report. I believe this bug to be fixed in youtube-dl 2015.01.05.1. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by BlackPropeller** [comment]
Great! I've updated and can confirm it's fixed! Thanks!
